package.json

```diff
@@ -34,22 +34,14 @@
     "@storybook/addon-storysource": "^3.4.12",
     "@storybook/react": "~3.4.12",
     "@storybook/storybook-deployer": "^2.0.0",
-    "@theforeman/vendor-dev": "^1.7.0",
+    "@theforeman/babel": "^3.2.0",
+    "@theforeman/env": "^3.2.0",
+    "@theforeman/vendor-dev": "^3.2.0",
     "argv-parse": "^1.0.1",
     "axios-mock-adapter": "^1.10.0",
-    "babel-cli": "^6.10.1",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^7.1.1",
-    "babel-plugin-dynamic-import-node": "^2.0.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-assign": "^6.8.0",
-    "babel-plugin-transform-object-rest-spread": "^6.8.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-register": "^6.9.0",
+    "babel-eslint": "^10.0.0",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "^8.0.0",
```

Comment:
`babel-loader` needs packaging PR.
(it is in the scope of `webpack`, not `babel`, as it's a webpack plugin)